core/src/test/java/org/springframework/security/authorization/AuthorityReactiveAuthorizationManagerTests.java

```diff
@@ -88,6 +89,15 @@ public void checkWhenHasAuthorityAndAuthorizedThenReturnTrue() {
 		assertThat(granted).isTrue();
 	}
 
+	@Test
+	public void checkWhenHasCustomAuthorityAndAuthorizedThenReturnTrue() {
```

Comment:
Can you also provide a test for when the `Authentication` has the wrong authorities but uses a custom implementation?

Reply:
Done.